src/screens/ChefScreen.tsx

```diff
@@ -1,67 +1,76 @@
 import React from "react";
-import { StyleSheet, Dimensions, View, Text, TouchableOpacity } from "react-native";
-import { StackNavigationProp } from '@react-navigation/stack';
-import { LoggedInParamList } from "../../types"
+import { useDispatch } from "react-redux";
+import {
+  StyleSheet,
+  Dimensions,
+  View,
+  Text,
+  TouchableOpacity,
+} from "react-native";
+import { StackNavigationProp } from "@react-navigation/stack";
+import { LoggedInParamList } from "../../types";
 import { colorPalette, shadowStyle } from "../constants/ColorPalette";
+import { firebaseApp } from "../constants/Firebase";
 
 const _screen = Dimensions.get("screen");
 export interface ChefScreenProps {
-    navigation: StackNavigationProp<LoggedInParamList, "ChefScreen">
+  navigation: StackNavigationProp<LoggedInParamList, "ChefScreen">;
 }
 
 export default function ChefScreen({ navigation }: ChefScreenProps) {
-    
-    // TODO: Make this page look like a profile page
-    return (
-        <View style={styles.container}>
-            <Text> Chefs Screen </Text>
-            <View style={styles.subContainer}>
-                <TouchableOpacity
-                    onPress={() => navigation.navigate("AboutUsScreen")}
-                >
-                    <Text>About Us</Text>
-
-                </TouchableOpacity>
-                <TouchableOpacity
-                    onPress={() => {
-
-                        // TODO: 
-                        // - Log Chef out
-                        // - Update global state
-                        navigation.navigate("LoginScreen")
-                    }}
-                >
-                    <Text>Logout</Text>
-
-                </TouchableOpacity>
-                <TouchableOpacity
-                    onPress={() => navigation.navigate("DeleteAccountScreen")}
-                >
-                    <Text>Delete Account</Text>
-
-                </TouchableOpacity>
-            </View>
-        </View>
-    )
+  // TODO: Make this page look like a profile page
+  const dispatch = useDispatch();
+  return (
+    <View style={styles.container}>
+      <Text> Chefs Screen </Text>
+      <View style={styles.subContainer}>
+        <TouchableOpacity onPress={() => navigation.navigate("AboutUsScreen")}>
+          <Text>About Us</Text>
+        </TouchableOpacity>
+        <TouchableOpacity
+          onPress={() => {
+            //Log out chef with firebase
+            firebaseApp
+              .auth()
+              .signOut()
+              .then(() => {
+                console.log("Signed out user successfully");
```

Comment:
Eventually we should be removing all console logs. It might make sense to move the dispatch and navigation in the ```.then()``` so that we await firebase